benchmarks/Jaeger.Benchmarks/Jaeger.Benchmarks.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What was changed in this line?

Reply:
My VS saves *.csproj files as UTF-8 BOM encoding, fixed.